frontend/src/views/SqlEditor/AsidePanel/DatabaseTree.vue

```diff
@@ -212,8 +214,31 @@ const handleSelectedKeysChange = (
     }
   }
 };
+
+function renderLabel({ option }: { option: TreeOption }) {
```

Comment:
Maybe we should define as arrow function instead of normal function here to unify the code format.